src/components/admin/statistics/BarGraphView.tsx

```diff
@@ -0,0 +1,32 @@
+import { Bar } from "react-chartjs-2";
+import React from "react";
+import { Chart, registerables } from "chart.js";
+import { Box, Heading } from "@chakra-ui/react";
+
+Chart.register(...registerables);
+
+interface IProps {
+    heading: string;
+    data: Record<string, number>;
+}
+
+const BarGraphView: React.FC<IProps> = props => (
+  <Box style={{ height: "1500px", width: "500px"}}>
+    <Heading style={{ textAlign: "center", fontSize: "25px" }}>School Year</Heading>
+      <Bar
+        data={{
+          labels: Object.keys(props.data),
+          datasets: [
+            {
+              label: "Number of applicants",
+              data: Object.values(props.data),
+              
```

Comment:
For this bar graph, can you order it so that it goes 1st year, 2nd year, 3rd year, etc?